cunumeric/deferred.py

```diff
@@ -463,7 +463,8 @@ def _zip_indices(
         task.add_scalar_arg(self.shape, (ty.int64,))
         for a in arrays:
             task.add_input(a)
-            task.add_alignment(output_arr.base, a)
+            if a.shape != ():
+                task.add_alignment(output_arr.base, a)
```

Comment:
I don't think this is necessary. Has this caused issues?

I am a little hesitant to skip alignment, even for seemingly trivial cases, as I don't know how the auto-partitioner will handle an under-constrained case like this. @magnatelee any comments?

Reply:
Actually, we don't need `if a.shape != ():` anymore. I will remove it in a minute